docker/apm-server/managed/docker-entrypoint.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e
```

Comment:
since we're relying on piping to `jq` we should use `-o pipefail`.  If we're going to use this longer term I'd suggest writing this not in bash